catroid/src/main/java/org/catrobat/catroid/content/actions/ArduinoSendDigitalValueAction.java

```diff
@@ -66,12 +66,7 @@ protected void begin() {
 					interpretationException);
 		}
 
-		if ((!restart)
-				&& (pinNumberInterpretation == 2)
-				|| (pinNumberInterpretation == 4)
-				|| (pinNumberInterpretation == 8)
-				|| (pinNumberInterpretation == 12)
-				|| (pinNumberInterpretation == 13)) {
+		if (!restart) {
 			this.pin = pinNumberInterpretation;
 			this.value = pinValueInterpretation;
 		}
```

Comment:
Alright!
I'll write a refactoring ticket that addresses those two arrays after all your PRs have been merged. Both contain the same information and always have to be in sync. So there is no need for two arrays anyway.